### PR TITLE
Port PR 558 (note about GELI passphrase on login) to angulargui

### PR DESCRIPTION
**Pull request by Mrt134**
- Build testing: broken for TrueNAS guide in this branch.

### CODE REVIEW

File: userguide/system.rst [hunk 1]
```diff
@@ -2681,13 +2681,20 @@ and described in
...
-   nodes to come up in standby mode. The |web-ui| shows an additional
-   :guilabel:`Force Takeover` button which can be used to force that
-   node to take control.
-
-.. tip:: The %brand% version of the :command:`ifconfig` command adds
-   two additional fields to the output to help with failover
-   troubleshooting: :samp:`CriticalGroup{n}` and :samp:`Interlink`.
+**Notes about High Availability and failovers:**
+
+Booting an HA pair with failover disabled causes both nodes to come up
```
Comment (wblock) [inline userguide/system.rst]: s/node/storage controllers/

File: userguide/system.rst [hunk 2]
```diff
@@ -2681,13 +2681,20 @@ and described in
...
+in standby mode. The |web-ui| shows an additional
+:guilabel:`Force Takeover` button which can be used to force that node
```
Comment (wblock) [inline userguide/system.rst]: s/node/storage controllers/

File: userguide/system.rst [hunk 3]
```diff
@@ -2681,13 +2681,20 @@ and described in
...
+to take control.
+
+The %brand% version of the :command:`ifconfig` command adds two
+additional fields to the output to help with failover troubleshooting:
+:samp:`CriticalGroup{n}` and :samp:`Interlink`.
+
+If both nodes reboot simultaneously, the GELI passphrase for an
```
Comment (wblock) [inline userguide/system.rst]: s/node/storage controllers/

File: userguide/system.rst [hunk 4]
```diff
@@ -2681,13 +2681,20 @@ and described in
...
+:ref:`encrypted <Encryption>` zpool must be entered at the |web-ui|
```
Comment (wblock) [inline userguide/system.rst]: Just "pool".

File: userguide/system.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/system.rst]: Changing "node" terminology to "storage controller" is blocked until the TrueNAS UI is updated to switch "node" to "storage controller"

File: userguide/system.rst (hunk 2 above)
Comment (Mrt134) [reply userguide/system.rst]: Changing "node" terminology to "storage controller" is blocked until the TrueNAS UI is updated to switch "node" to "storage controller"

File: userguide/system.rst (hunk 3 above)
Comment (Mrt134) [reply userguide/system.rst]: Changing "node" terminology to "storage controller" is blocked until the TrueNAS UI is updated to switch "node" to "storage controller"

**Pull request merged**